### PR TITLE
Add roadmap section to community pages

### PR DESCRIPTION
**Pull request by ryyppy**

### CODE REVIEW

File: pages/community/roadmap.mdx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+- Stable (LTS) & Unstable release channel
+- Always use stable releases for libraries!
+- You may want to use unstable if you want to test new features in your applications
+- Unstable will be useful to prepare your codebase / libraries for an upcoming stable release
+
+## Definition Stable / LTS / Unstable
+
+We will follow a similar approach to the NodeJS project and consider even version numbers to be stable LTS (Long Term Support) releases (v10, v12, v14,...), and uneven version numbers as unstable / experimental (v11, v13, v15,...) releases.
+
+The goal of the new release system is to have a slower and easier to manage upgrading experience for our users, with a proper grace-period to discuss and refine feature proposals.
```
Comment (peterpme) [inline pages/community/roadmap.mdx:26]:
```suggestion
The goal of the new release system is to have a slower and easier to manage upgrade experience for our users, with a proper grace-period to discuss and refine feature proposals.
```

File: pages/community/roadmap.mdx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+- A stable release will eventually require some initial migration. We will provide proper upgrade guides between each stable version.
+- During active development, there are no breaking changes within the compiler (no changes in runtime representation, behavior of the build system, configuration names etc.). 
+- No removal of language features
+- During the active development phase, there may be non-breaking minor releases (e.g. v10.1.0). For example:
+  - Opt-in configuration flag `stdlib-future` to disable OCaml related stdlib code (for writing future proof code)
+  - Stdlib additions / Library enhancements
+- Regular bugfix releases for the compiler, build system, stdlib, etc.
+- Docs will be synced up with the latest stable release
+- Whenever a new stable release is being published, the previous stable release reaches end-of-life (EOL), and therefore will only receive bugfix updates in a best effort service. After 1 year LTS, we expect the retired version to be pretty stable though, so in practise, users will be able to stay on that version indefinitely until they decide to upgrade to the newest stable
```
Comment (peterpme) [inline pages/community/roadmap.mdx:42]:
```suggestion
- Whenever a new stable release is published, the previous stable release reaches end-of-life (EOL), and therefore will only receive bugfix updates in a best effort service. After 1 year LTS, we expect the retired version to be pretty stable though, so in practise, users will be able to stay on that version indefinitely until they decide to upgrade to the newest stable
```

File: pages/community/roadmap.mdx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+**Planned Changes for v10:**
+
+- Add M1 support
+- Add support for unicode strings (e.g `"Hallöle"` will work like in JavaScript)
+- (breaking) Remove OCaml’s Format module (should have no impact on the majority of ReScript users)
+- Infix operator support (only for a dedicated set of math operators `+`, `-`, `-.`, `/`, `%`, etc.)
+- Docstring syntax support (`/** */` instead of `@ocaml.doc("My docstring")`)
+- Convert stdlib files (`Js`, `Belt`, etc) to ReScript syntax
+- Some other less prominent enhancements (will be mentioned in the Changelog later on)
+- We will eventually announce additional breaking changes in late 2021 if necessary
```
Comment (Minnozz) [inline pages/community/roadmap.mdx:78]: Maybe these should link to the corresponding GitHub issues/epics? That way it is easier to look up the current status of these changes and/or contribute to them.

**Comment by bobzhang** [comment]
lgmt, maybe should add we will maintain 9.1 bug fix release until the end of this year?

**Comment by ryyppy** [comment]
still want to rephrase the term LTS to "maintenance phase" and restructure the document a little bit.

Yes we can add a note on 9.1

File: pages/community/roadmap.mdx (hunk 3 above)
Comment (ryyppy) [reply pages/community/roadmap.mdx]: added

Comment (Minnozz) [reply pages/community/roadmap.mdx]: Awesome!

File: pages/community/roadmap.mdx [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+- (breaking) Remove OCaml’s Format module (should have no impact on the majority of ReScript users; see [discussion](https://forum.rescript-lang.org/t/what-stable-type-system-features-we-are-going-to-preserve-from-ocaml/1251))
+- Infix operator support (only for a dedicated set of math operators `+`, `-`, `-.`, `/`, `%`, etc.; see the [tracking issue](https://github.com/rescript-lang/syntax/issues/440))
+- Docstring syntax support: `/** */` instead of `@ocaml.doc("My docstring")` ([tracking issue](https://github.com/rescript-lang/syntax/issues/439))
+- Convert stdlib files (`Js`, `Belt`, etc) to ReScript syntax ([tracking issue](https://github.com/rescript-lang/rescript-compiler/issues/4872))
+- Make `bsconfig.json` json compliant ([Proposal](https://forum.rescript-lang.org/t/proposed-changes-make-bsconfig-json-standard-compilant/1859))
+- Some other less prominent enhancements (will be mentioned in the Changelog later on)
+- (Non exhaustive list of changes; goals may be adapted)
```
Comment (bobzhang) [inline pages/community/roadmap.mdx:89]:
```suggestion
```

File: pages/community/roadmap.mdx [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+
+Please be aware that our roadmap will only discuss major changes, and might be adapted during planning phases.
+
+All roadmap related discussions, proposals and changes for specific roadmap tasks can either be found in the [ReScript Forum](https://forum.rescript-lang.org), or in dedicated Github PRs / issues.
+
+## Release Plan (v10 - v15)
+
+<Image withShadow={false} src="/static/img/roadmap_2021.jpg" caption="Release cycle for the upcoming versions"/>
+
+### Stable Release v10
```
[the remark this answers is not in the document]
Comment (ryyppy) [reply pages/community/roadmap.mdx:78]: naaah, we won't put that in the header that looks really ugly.. i think ppl will not be skipping on the last point in the change list

**Pull request merged**